### PR TITLE
PR: Client only version

### PR DESCRIPTION
**Pull request by zeljkotutic**

### CODE REVIEW

File: src/commands/commands.html [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
-  </head>
+</head>
+
+<body>
+  
+</body>
 
-  <body></body>
-</html>
+</html>
```
Comment (flokli) [inline src/commands/commands.html:20]: Please add a trailing newline.

Comment (flokli) [reply src/commands/commands.html:20]: Done.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
[the remark this answers is not in the document]
Comment (flokli) [reply package.json:1]: Done, please mark resolved.

File: manifest-staging.xml [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
+  <Description DefaultValue="Wire meeting integration for Outlook"/>
+  <IconUrl DefaultValue="https://outlook.integrations.zinfra.io/client/assets/icon-64.png"/>
+  <HighResolutionIconUrl DefaultValue="https://outlook.integrations.zinfra.io/client/assets/icon-128.png"/>
+  <SupportUrl DefaultValue="https://outlook.integrations.zinfra.io/client/support.html"/>
   <AppDomains>
-  <!-- This is needed to avoid CORS, outlook side -->
-    <AppDomain>https://localhost:3000</AppDomain>
-    <AppDomain>https://app.wire.com</AppDomain>
     <AppDomain>https://staging-nginz-https.zinfra.io</AppDomain>
     <AppDomain>https://wire-webapp-edge.zinfra.io/</AppDomain>
```
[the remark this answers is not in the document]
Comment (flokli) [reply manifest-staging.xml]: This now become AUTHORIZE_HOST, but is still an open question.

File: manifest-staging.xml [hunk 4]
```diff
@@ -104,13 +101,13 @@
           </ExtensionPoint>
 
           <!-- Enable launching the add-in on the included event. -->
-          <ExtensionPoint xsi:type="LaunchEvent">
+          <!-- <ExtensionPoint xsi:type="LaunchEvent">
```
[the remark this answers is not in the document]
Comment (flokli) [reply manifest-staging.xml]: Done, please mark resolved.

File: manifest-staging.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (zeljkotutic) [reply manifest-staging.xml]: You can see example in offical repo:
https://github.com/OfficeDev/generator-office/blob/master/src/app/templates/hosts/outlook/manifest.xml#L32


 We're navigating to AUTHORIZE_URL (it's webapp URL in our case), so I'll leave this. On one other source I saw that we need also domains that are used in fetch calls, however it seems it's not like that, so I'm removing API_HOST. I'll double check this also later (adding on TODO), however will merge it like this as @dkovacevic wants to test new approach ASAP.

File: src/authorize/authorize.ts [hunk 5]
```diff
@@ -41,11 +46,7 @@ const generateCodeVerifier = async (): Promise<string> => {
 
 const generateCodeChallenge = async (codeVerifier: string): Promise<string> => {
   const hash = CryptoJS.SHA256(codeVerifier);
-  const base64Url = hash
-    .toString(CryptoJS.enc.Base64)
-    .replace('+', '-')
-    .replace('/', '_')
-    .replace(/=+$/, '');
+  const base64Url = hash.toString(CryptoJS.enc.Base64).replace("+", "-").replace("/", "_").replace(/=+$/, "");
```
[the remark this answers is not in the document]
Comment (zeljkotutic) [reply src/authorize/authorize.ts]: Sure, thanks for suggestion that this is also available.

**Pull request merged**